### ISSUE TITLE
Explore using Linux Subsystem for Windows 10 users

### ISSUE THREAD

**Issue by Holyoke** [railsbridge/docs#580]
Currently the instructions for Windows users include Rails Installers, which is not behaving well for users on Windows 10. Normally I'd introduce a virtual IDE with Cloud9, but unfortunately they now require a credit card for sign up. 

I'd like remark on newer options for development on Windows 10:

## Using Linux Subsystem  
Currently this seems to be a newer option for Windows 10 users. Windows 10 now has a native BASH environment, meaning that it has a Terminal interface. Here's a tutorial: https://gorails.com/setup/windows/10. The instructions largely has the W10 user opt-in for beta updates with their Microsoft account, which enables the Linux subsystem. 

This way, W10 users should have a native and direct implementation of Ruby and Rails. All in all, W10's new Linux subsystem enables a native Linux environment within.

ETA: ~45 minutes for student

## Using Nitrous 
Nitrous is a Virtual IDE like Cloud9, and it has no credit card required for it's free tier. IT allows 50 hours of project time with 2 project limit for free, so it's ideal for a coding Workshop. I will start recommending this since it's the same experience as Cloud9 as well. 

This is the most minimal setup required but like Cloud9, it abstracts the development environment away from the student. 

ETA: ~10 minutes for student

## Creating a Live CD/DVD/USB using Linux
Alternatively, I wonder if there was any talk of having a live Linux distro. I've had cases where the Rails environment isn't working at all on a Windows machine and virtual IDE's aren't easily available. Having a Live Linux distro with Ubuntu could mean having any PC become a web development machine just by booting a flash drive. 

This would require getting a clean image and putting it on a 8gb flash drive. I would like to have this option available next time and have other people consider it in dire cases. I'd recommend Ubuntu since that's the most popular and consumer friendly distro. 

ETA: ~5 minutes for student

### tl;dr

I'd recommend NItrous as the new virtual IDE for windows users. I'd like to explore instructions for installing Linux Subsystem for W10 users so they can write rails natively.

**Comment by wuworkshop** [comment]
So apparently Nitrous.io shut down its service around November 2016 - https://techcrunch.com/2016/10/31/cloud-development-platform-nitrous-io-shuts-down/

> Cloud9, but unfortunately they now require a credit card for sign up

Wow, didn't even know about that and they don't make it obvious when a new user initially signs up. I would have to agree with this user from their Community forum - https://community.c9.io/t/how-about-you-tell-people-you-require-credit-card-details-for-a-free-account-before-they-waste-time-filling-registration-forms/13123 

Also just realized they were acquired by Amazon last year - https://c9.io/blog/great-news/ =/

Would really love to know if a live Linux USB drive method has been attempted. These cloud-based IDEs don't seem too reliable...

**Comment by maxjacobson** [comment]
@wuworkshop ScriptEd uses Cloud9 and has some kind of deal where the students don't pay. I wouldn't be surprised if RailsBridge could get a similar deal.

**Comment by wuworkshop** [comment]
@maxjacobson You're probably referring to Cloud9's Education plan. https://c9.io/pricing

I noticed yesterday that the Rails Girls Guide suggest https://codenvy.io/ - http://guides.railsgirls.com/install#using-a-cloud-service

It does appear to still offer a free plan for developers - https://codenvy.com/product/pricing/
